### ISSUE TITLE
Service HTTP Template does not recognize source IP prefix length

### ISSUE THREAD

**Issue by danpacheco1** [F5Networks/f5-appsvcs-extension#345]
### Environment
 * Application Services Version: 3.22.1
 * BIG-IP Version: 14.1.2.6

### Summary
My declaration produces the following error: 
"0107028a:3: The source address (10.0.0.0%110) for virtual server (/as3-partition/as3-app/as3-vs-Redirect-) must have a prefix length."
Despite the source prefix length being specified in the declaration. I tried with with and without the route domain and the issue persisted.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
   "class": "AS3",
   "action": "deploy",
   "persist": true,
   "declaration": {
      "class": "ADC",
      "schemaVersion": "3.0.0",
      "id": "abc123",
      "label": "labels the declaration",
      "updateMode": "selective",
      "remark": "another comment field",
      "as3-partition": {
         "class": "Tenant",
         "defaultRouteDomain": 110,
         "as3-app": {
            "class": "Application",
            "template": "generic",
            "as3-vs": {
               "class": "Service_HTTPS",
               "virtualAddresses": [
                  [
                  "7.7.7.7/32",
                  "10.0.0.0/8"
                  ]
                ],
               "virtualPort": 8080,
               "serverTLS": "as3-client-ssl",
               "pool": "web_pool"
            },
            "as3-client-ssl": {
                "class": "TLS_Server",
                    "certificates": [
                    {
                        "certificate": "as3-cert"
                    }
                ],
                "cacheTimeout": 1234
            },
            "as3-cert": {
                "class": "Certificate",
                "remark": "in practice using a passphrase is recommended",
                "certificate": {
                    "bigip": "/Common/default.crt"
                },
                "privateKey": {
                    "bigip": "/Common/default.key"
                }
            },
            "web_pool": {
               "class": "Pool",
               "monitors": [
                  "http"
               ],
               "members": [{
                  "servicePort": 80,
                  "serverAddresses": [
                     "192.0.1.10",
                     "192.0.1.11"
                  ]
               }]
            }
         }
      }
   }
}

```

2. Observe the following error response:
```json
{
"code": 422,
"message": "declaration failed",
"response": "0107028a:3: The source address (10.0.0.0%110) for virtual server (/as3-partition/as3-app/as3-vs-Redirect-) must have a prefix length.",
"host": "localhost",
"tenant": "as3-partition",
"runTime": 3913
}
```

### Expected Behavior
status 200 and a new vs created with a source of 10.0.0.0/8

### Actual Behavior
error message 0107028a:3: The source address (10.0.0.0%110) for virtual server (/as3-partition/as3-app/as3-vs-Redirect-) must have a prefix length.",

**Comment by dstokesf5** [comment]
Thank you for reporting this bug. It is in our product backlog as AUTOTOOL-1946.

**Comment by dstokesf5** [comment]
This was resolved in the 3.24.0 release.
